lc-hax/Scripts/Commands/GrabCommand.cs

```diff
@@ -37,7 +37,7 @@ Transform parentObject
         string itemName
     ) {
         Dictionary<string, GrabbableObject> grabbableObjects =
-            HaxObjects.Instance?.GrabbableObjects?
+            Helper.Props?
```

Comment:
`Props` would include furniture/buildings so I'd rather we just stick to a more concise name like `GrabbableObjects` or `Grabbables`.

Reply:
All done